frontend-project/package.json

```diff
@@ -18,7 +18,7 @@
     "lint:style": "stylelint --fix \"src/**/*.less\" --syntax less",
     "prettier": "prettier -c --write \"**/*\"",
     "start": "umi dev",
-    "start:public-api": "cross-env API_URL=\"https://dev.small-eod.siecobywatelska.pl/\" yarn start",
+    "start:public-api": "cross-env API_URL=\"http://localhost:8000\" yarn start",
```

Comment:
Z założenia `start:public-api` miało służyć do wskazywania na publiczne ogólnodostępny back-end, aby rozwijać front-end bez dotykania front-endu. Taki skrót. Jeżeli chcesz używać lokalnego API to używasz po prostu `start`.

Reply:
Przypadkowo dodałem. My bad :disappointed: